src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -625,66 +612,12 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                         WellKnownMember.System_Security_UnverifiableCodeAttribute__ctor));
                 }
             }
-
-            if (UtilizesNullableReferenceTypes)
-            {
-                // PROTOTYPE(NullableReferenceTypes): Support NetModule.
-                if (compilation.Options.OutputKind != OutputKind.NetModule)
-                {
-                    AddSynthesizedAttribute(ref attributes, compilation.TrySynthesizeAttribute(WellKnownMember.System_Runtime_CompilerServices_NullableAttribute__ctor));
```

Comment:
>AddSynthesizedAttribute(ref attributes, compilation.TrySynthesizeAttribute(WellKnownMember.System_Runtime_CompilerServices_NullableAttribute__ctor)); [](start = 20, length = 149)

Are we no longer emitting this attribute on the module?

Reply:
Thanks. We should generate `[module: Nullable]`. I've added a commented out `EmitAttribute_Module` test for now.